test/parallel_api/experimental/for_loop_induction.pass.cpp

```diff
@@ -60,11 +62,11 @@ test_body_induction(Policy&& exec, Iterator /* first */, Iterator /* last */, It
         });
 
     EXPECT_TRUE(lval_ind == n, "wrong result of induction");
-    EXPECT_TRUE(rval_ind == ind_init, "wrong result of induction");
```

Comment:
Seems like the static analysis tool is right. We should not touch moved objects.

I believe the whole idea of rvalue parameters being used it to prove that induction could work with rvalues. On the other hand we are saying that now this rvalue (after move) belongs for for_loop and can be read only within for_looop body but not the outside. That is the similar situation with capturing the object to a lambda by value, you cannot access it outside of the lambda.

Proposal is to leave the code as before PR and remove check on line 63 and 67.